skiko/src/iosMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.ios.kt

```diff
@@ -224,6 +224,7 @@ internal class MetalLayer : CAMetalLayer {
             this.backgroundColor =
                 CGColorCreate(CGColorSpaceCreateDeviceRGB(), it.addressOf(0))
         }
+        this.framebufferOnly = false
```

Comment:
How will it affect FPS for UI that doesn't use blending?

Could you check on some not simple example (run1 on iOS, for example)?

Reply:
Sure I'll check, I'd say some (<1ms) drop is expected. But anyway correctness is more important than that

Reply:
> correctness is more important than that

Sure, but if it would significantly affect FPS, we have alternatives.

Reply:
During my tests on iPad mini 2, I do not see the difference - frame time avg is bouncing about the same values